.github/workflows/pr.yaml

```diff
@@ -54,11 +54,65 @@ jobs:
         with:
           go-version-file: 'go.mod'
 
+      - name: Check for outdated bundle
+        id: bundle-diff-checker
+        # Lot of debate (https://github.com/janus-idp/operator/pull/195) whether this should be a warning or an error.
+        # For now, this is will be warning + a comment on the PR if manifests are outdated. This way, PR authors/maintainers can be aware of that fact.
```

Comment:
I guess a comment is at least visible to the PR review process, but if action is required to change the PR before merging, then why not FAIL the action and FORCE the developer to fix their PR, instead of just suggesting what needs doing via a comment?